tools/depth-quality/depth-quality-model.h

```diff
@@ -143,20 +149,39 @@ namespace rs2
 
             callback_type callback;
 
-            void set_ground_truth(float gt)
+            void set_ground_truth(int gt)
             {
                 std::lock_guard<std::mutex> lock(_m);
-                _ground_truth = gt;
+                _ground_truth_mm = gt;
                 _use_gt = true;
             }
-            void disable_ground_truth() 
+
+            void set_plane_fit(bool found)
             {
                 std::lock_guard<std::mutex> lock(_m);
-                _use_gt = false; 
+                if (_plane_fit != (int)found)
+                {
+                    // Affects the visibility of plane-fit-dependent metrics
+                    _plane_fit = found;
+                    for (auto&& plot : _plots)
+                    {
+                        if (plot->requires_plane_fit())
+                            plot->visible(found);
+                    }
+                }
             }
-            float get_ground_truth() const { std::lock_guard<std::mutex> lock(_m); return _ground_truth; }
 
-            void reset() {
+            void disable_ground_truth()
+            {
+                std::lock_guard<std::mutex> lock(_m);
+                _use_gt = false;
+                _ground_truth_mm = 0;
+            }
+            std::tuple<int, int> get_inputs() const { std::lock_guard<std::mutex> lock(_m); return std::make_tuple(_ground_truth_mm, _plane_fit); }
+
+            void reset()
+            {
+                _plane_fit = -1;
                 rs2::frame f;
                 while (_frame_queue.poll_for_frame(&f));
```

Comment:
`while (_frame_queue.poll_for_frame(&f));` is possibly infinite, consider adding another condition

Reply:
Actually, queue.flush is required